docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -24,6 +24,12 @@ prefer. Vagrant is available in Fedora 21 and newer. Follow these steps:
    
       $ sudo yum install ansible vagrant-libvirt
 
+#. Optionally, install `vagrant-cachier <http://fgrehm.viewdocs.io/vagrant-cachier>`_. It will cache
+   the packages that are downloaded during provisioning on your host so that the next time you
+   provision you will save some time and bandwidth::
+
+      $ vagrant plugin install vagrant-cachier
```

Comment:
This depends on the `ruby-devel`, `gcc-c++`, and `libvirt-devel` (and maybe some other packages I already had).